.circleci/setup-system

```diff
@@ -8,13 +8,15 @@ set -eu
 
 sudo apt-get update
 
+# Add repo to install a sufficiently-recent version of node (for atdts tests)
+curl -fsSL https://deb.nodesource.com/setup_16.x | sudo -E bash -
+
 # This includes compilers and libraries to support the various target
 # languages of atd.
 #
 sudo apt-get install -y \
   default-jdk \
   nodejs \
-  npm \
```

Comment:
npm comes with nodejs when using the nodesource repo?

Reply:
There was a package conflict so that's what inferred but now we're getting another error: missing `js` command. I'll figure this out.

Reply:
idk, maybe we don't need the `js` command anyway. I confirm that `npm` was installed.

Reply:
Calling `node` instead of `js` works.